contributors/devel/sig-architecture/api_changes.md

```diff
@@ -922,7 +922,10 @@ doing!
 Check out the [E2E docs](../sig-testing/e2e-tests.md) for detailed information about how to
 write end-to-end tests for your feature.
 Make sure the E2E tests are running in the default presubmits for a feature that
-is being promoted to Beta (enabled by default).
+is being promoted to Beta (disabled by default, make sure to enable with flag).
+This behavior changed with Kubernetes v1.24 (see the 
+[KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-architecture/3136-beta-apis-off-by-default/README.md) 
+for more details) and only applies to new beta API introduced after v1.24.
```

Comment:
I'd simplify this and say "Make sure the E2E tests are running in the default presubmits for a feature / API that is enabled by default."